app/controllers/geckoboard_api/application_controller.rb

```diff
@@ -1,5 +1,4 @@
 class GeckoboardApi::ApplicationController < ActionController::Base
-  protect_from_forgery prepend: true, with: :exception
```

Comment:
Out of scope but this endpoint and other geckoboard controllers should be `skip_forgery_protection` as they are either used by the geckoboard API (which cannot provide a CSRF authentication token or are get requests which do not use CSRF protection anyway `rails routes | grep geckoboard`
```

Reply:
see https://stackoverflow.com/questions/9446946/csrf-protection-with-http-get-requests-in-rails